Directory.Packages.props

```diff
@@ -10,10 +10,9 @@
     <PackageVersion Include="Microsoft.Extensions.Configuration.CommandLine" Version="7.0.0" />
     <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0" />
     <PackageVersion Include="Microsoft.Extensions.DependencyModel" Version="7.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Logging" Version="7.0.0" />
     <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.0" />
     <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="all" />
-    <PackageVersion Include="ConcurrentHashSet" Version="1.3.0" />
-    <PackageVersion Include="DSharpPlus.VoiceNext.Natives" Version="1.0.0" />
```

Comment:
interactivity and voicenext still require those, as far as i'm aware

Reply:
i don't think voicenext even works at this stage /j
but yes, you're right

Reply:
> interactivity and voicenext still require those, as far as i'm aware

They're duplicates